content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -301,6 +301,69 @@ already attempted to run with the bad configuration.
 StatefulSet will then begin to recreate the Pods using the reverted template.
 
 
+## PersistentVolumeClaim retention
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+The optional `.spec.persistentVolumeClaimRetentionPolicy` field controls if
+and how PVCs are deleted during the lifecycle of a StatefulSet. You must enable the
+`StatefulSetAutoDeletePVC` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+to use this field. Once enabled, there are two policies you can configure for each
+StatefulSet:
+
+`whenDeleted`
+: configures the volume retention behavior that applies when the StatefulSet is deleted
+
+`whenScaled`
+: configures the volume retention behavior that applies when the replica count of
+  the StatefulSet   is reduced; for example, when scaling down the set.
+  
+For each policy that you can configure, you can set the value to either `Delete` or `Retain`.
+
+`Delete`
+: The PVCs created from the StatefulSet `volumeClaimTemplate` are deleted, for
+  each Pod affected by the policy. With the `whenDeleted` policy, as all Pods are
+  deleted when the Statefulset is deleted, all PVCs from the `volumeClaimTemplate`
```

Comment:
This clause `as all Pods are deleted when the Statefulset is deleted` is a bit difficult to follow.  Might suggest

> With the `whenDeleted` policy, each PVC from the `volumeClaimTemplate` will be deleted after the corresponding pod is deleted. [ed: you may want to describe exactly the ordering here - does is the ordering dependent on the foreground policy?  Does the controller wait until it observes the pod is fully deleted?  These help clarify how someone knows what to debug when it goes wrong].  

(see my editor note, I think we should clarify the time order in words of what waits before what)

Reply:
Improved the wording, or tried to, in the context of your comment below.